solidity/ecdsa/contracts/WalletRegistryGovernance.sol

```diff
@@ -165,7 +168,27 @@ contract WalletRegistryGovernance is Ownable {
             "New random beacon address cannot be zero"
         );
 
-        walletRegistry.upgradeRandomBeacon(_newRandomBeacon);
+        walletRegistry.upgradeRandomBeacon(IRandomBeacon(_newRandomBeacon));
+    }
+
+    /// @notice Initializes the Wallet Owner's address.
+    /// @dev Can be called only by the contract owner. It can be called only if
+    ///      walletOwner has not been set before. It doesn't enforce a governance
+    ///      delay for the initial update. Any subsequent updates should be performed
+    ///      with beginWalletOwnerUpdate/finalizeWalletOwnerUpdate with respect
+    ///      of a governance delay.
+    /// @param _walletOwner The Wallet Owner's address
+    function initializeWalletOwner(address _walletOwner) external onlyOwner {
+        require(
+            address(walletRegistry.walletOwner()) == address(0),
+            "Wallet Owner already initialized"
+        );
```

Comment:
It should be possible to update the wallet owner, let's not handcuff ourselves. It may happen we may need to upgrade `Bridge` contract address.

Reply:
There is a standard path for `walletOwner` update with `beginWalletOwnerUpdate` and `finalizeWalletOwnerUpdate`.
`initializeWalletOwner` is here as we cannot set the initial value in the `WalletRegistry` constructor due to the cyclic dependency between contracts.

Reply:
FWIW, we will have to revisit the initialization of this contract. We have more logic in `WalletRegistry` now and I think it should be deployed behind a transparent upgradeable proxy. If so, some constructor parameters will need to be moved to initialize function.

Reply:
Sure, let's just make sure we register it in the backlog.

Reply:
https://github.com/keep-network/keep-core/issues/2861 and https://github.com/keep-network/tbtc-v2/issues/161.